scenario/features/chaincodeevents.feature

```diff
@@ -53,3 +53,18 @@ Feature: Chaincode event listening
         And I set the transaction arguments to ["close", "after"]
         And I invoke the transaction
         Then I should receive a chaincode event named "close" with payload "after" on "listener2"
+
+    Scenario: Checkpoint of chaincode events
+        Given I create a checkpointer
+        And I use my checkpointer to listen for chaincode events from basic
```

Comment:
To better fit the language used in other steps (e.g. "I invoke **the** transaction"):
```suggestion
        And I use the checkpointer to listen for chaincode events from basic
```